netengine/backends/snmp/base.py

```diff
@@ -57,6 +59,20 @@ def _octet_to_mac(self, octet_mac):
             )
         return mac_address
 
+    def _ascii_blocks_to_ipv6(self, ascii_string):
```

Comment:
Are you sure this is ASCII representation of IPv6? I am not sure about this.

Reply:
Yes, these are the ascii values converted into hexadecimal by this function. You can look at [this chart](https://www.garykessler.net/library/ascii.html) for example.
For example if an ip address is `fe80::....` then we should see something like `254.128....` in the oid